synapse/tests/test_model_inet.py

```diff
@@ -303,3 +303,16 @@ def test_model_inet_follows(self):
     def test_model_inet_ipv4_raise(self):
         with s_cortex.openurl('ram:///') as core:
             self.raises( BadTypeValu, core.formTufoByProp, 'inet:ipv4', 'lolololololol' )
+    def test_model_inet_urlfile(self):
+        with s_cortex.openurl('ram:///') as core:
+            url = 'HTTP://visi:hehe@www.vertex.link:9999/'
+            fguid = 32 * 'a'
+            node = core.formTufoByProp('inet:urlfile', (fguid, url), **{'seen:min':0, 'seen:max':0})
+
+            self.eq( node[1].get('inet:urlfile:url:port'), 9999 )
```

Comment:
these properties ( underneath inet:urlfile:url: ) are only populated in this cortex because it doesn't have model enforcement enabled ( under normal circumstances, only the properties actually declared within the form are populated on the node, even if they are sub props of the type ) so it probably makes sense to remove these and rely on the type tests for URL to ensure they get broken out as they should.  The behavior is basically to allow the model to have a secondary property of type inet:url and not end up with a million sub-properties unless they are relevant enough to declare into the model :)